src/selectivity-multiple.js

```diff
@@ -441,6 +467,7 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
 
         if (this.options.showDropdown !== false) {
             this.open();
+            this.$('.selectivity-multiple-input-container').toggleClass("open", true);
```

Comment:
If you implement this logic in the base class, then this statement can also be moved into the `open()` method, which I believe would be cleaner.